packages/patternfly-4/react-docs/src/components/example/liveDemo.js

```diff
@@ -44,7 +46,7 @@ const scopePlayground = {
 const transformCode = code => {
   try {
     // LiveEditor doesn't work properly with these so need to remove
-    code = code.replace(/^import(.|\s)*?;$/gm, '');
+    code = code.replace(/^import(.|\s)*?;$/gm, '');1
```

Comment:
is the "1" just a random typo here? it seems to be ;)

Reply:
thanks :)